kibbeh/src/modules/layouts/ElectronHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import isElectron from "is-electron";
+import React from "react";
+import { WinMaximizeIcon, WinCloseIcon, WinMinimizeIcon } from "../../icons";
+import { Button } from "../../ui/Button";
+
+let ipcRenderer: any = undefined;
+if (isElectron()) {
+  ipcRenderer = window.require("electron").ipcRenderer;
+}
+
+export function ElectronHeader() {
+  return (
+    <div className="electron-header z-50">
```

Comment:
`user-select: none` is added to custom headers as a norm as it is not part of the normal UI but part of the application

Reply:
Also adding `-webkit-app-region: drag;` would allow us to drag the window wherever we want.